### DISCUSSION TITLE
terminate immediately without any error

### DISCUSSION THREAD

**Discussion by sagadmoh** [openai/whisper#2274] [Q&A]
I ran this command in Windows 10 x64:

whisper audio.wav --model tiny.en --fp16 False

After a few seconds, it will terminate without any error.
My computer's CPU is an AMD Phenom(tm) II X4 955 Processor   3.20 GHz

**Comment by glangford** [comment]
There isn't enough information here to say what the problem is. Some ideas:

* what happens if you enter `whisper --help`
* do you have `ffmpeg` installed correctly? enter `ffmpeg -version`
* confirm that `audio.wav` exists
* try with a different audio file

**Reply by sagadmoh** [reply]
Yes, I have installed Whisper recently. Actually, I installed it on my laptop before and it worked without any issues. However, on my PC, the process seems to have finished without producing any result or error. I am confident in the installation because I did it successfully on my laptop. I suspect the issue might be related to my PC's older CPU or other hardware problems, but I can't figure out where the problem is or how to find it.
